metadata.rb

```diff
@@ -728,6 +728,17 @@
 :description => "url for git sourcecode for tensorflow",
 :type => "string"
 
+attribute "tensorflow/custom_url",
+          :description => "User-supplied URL for the tensorflow .whl binaries to be installed.",
+          :type => "string"
+
```

Comment:
Since Karamel 0.5, this is not required anymore. Having the attribute in tensorflow-chef is enough.